src/db/index.ts

```
@@ -0,0 +1,38 @@
+import postgres from 'postgres'
+import { config } from '@/config'
+import { logger } from '@/utils/logger'
+
+export const clientPostgres = () => postgres(config().DATABASE_URL)
+
+export const getClient = () => {
+  return clientPostgres()
+}
+
+// eslint-disable-next-line @typescript-eslint/no-empty-object-type
+const db = (client?: postgres.Sql<{}>) => {
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)